airbyte-commons/src/main/java/io/airbyte/commons/json/Jsons.java

```diff
@@ -189,6 +198,25 @@ public static int getIntOrZero(final JsonNode json, final List<String> keys) {
     return optional.map(JsonNode::asInt).orElse(0);
   }
 
+  public static JsonNode flattenConfig(final JsonNode config) {
```

Comment:
why "config"? it is in a common library that manipulates json objects, nothing to do with config.

Reply:
this function is also oddly specific to be in a common library. 

I would expect flatten to behave the following way:
```
{
  'a': 1,
  'b': {
    'c': 3
  }
}
=>
{
  'a': 1,
  'b.c': 3
}
```

Reply:
yes, thanks for pointing it out, i'll move it out to a more specific oauth class